Makefile

```diff
@@ -14,6 +14,10 @@ end-to-end-test:
 	sudo docker build -f ./test/end-to-end/Dockerfile -t weld/end-to-end:latest ./test/end-to-end/
 
 	make -C welder-deployment/ repos
+	git clone ../welder-web
```

Comment:
The command sequence you need is:

git clone https://github.com/weldr/welder-deployment - clones the welder-deployment repo under ./welder-deployment
git clone ../welder-web welder-deployment/welder-web - clones (aka copies) the current checkout under welder-deployment/welder-web
make -C welder-deployment/ repos - will git clone bdcs and bdcs-api-rs repositories. Will not clone welder-web because the directory already exists.